java/src/jmri/jmrit/display/layoutEditor/PositionablePoint.java

```diff
@@ -113,40 +128,6 @@ public int getType() {
         return type;
     }
 
-    /**
```

Comment:
Deprecated methods that were in a production release should be retained for at least two production releases after it is deprecated. (If a method was in 4.8, and deprecated in 4.9.x, it needs to be retained in 4.10 and 4.12.)

Reply:
This (identical) method is in the base class so I removed it here. (shadow method)